http-add-on/Chart.yaml

```diff
@@ -10,7 +10,7 @@ kubeVersion: ">=v1.23.0-0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.1
+version: 0.4.2
```

Comment:
Please revert this change. We update the version during releases

Reply:
Ah, thanks!  I will revert that and add to the comment stating such, as the comment led me to believe I had to increment it myself.